mlir/lib/Dialect/Vector/Transforms/VectorTransforms.cpp

```diff
@@ -2049,6 +2048,12 @@ void mlir::vector::populateSinkVectorBroadcastPatterns(
       patterns.getContext(), benefit);
 }
 
+void mlir::vector::populateReoderVectorTransposePatterns(
```

Comment:
Above: should `populateSinkVectorBroadcastPatterns()` be `populateReorderVectorBroadcastPatterns()` for consistency?